popup/popup.js

```diff
@@ -276,15 +276,20 @@ function createStyleElement({
     });
     styleName.checkbox = checkbox;
     styleName.appendChild(document.createTextNode(' '));
+
     const config = $('.configure', entry);
+    config.onclick = handleEvent.configure;
     if (!style.usercssData && style.updateUrl && style.updateUrl.includes('?') && style.url) {
+      config.href = style.url;
       config.target = '_blank';
       config.title = t('configureStyleOnHomepage');
       config.dataset.sendMessage = JSON.stringify({method: 'openSettings'});
       $('use', config).attributes['xlink:href'].nodeValue = '#svg-icon-config-uso';
+    } else if (style.usercssData && Object.keys(style.usercssData.vars || {}).length === 0) {
```

Comment:
It doesn't hide the config button if the style is installed from USO and doesn't have configurable options.

Reply:
LOL Funny, I don't have any styles installed from USO anymore, so I forgot to check.